prometheus/promdb/promdb_test.go

```diff
@@ -0,0 +1,241 @@
+package promdb
+
+import (
+	"database/sql"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
```

Comment:
dto?

Reply:
this import was just pulled from one of their files. I can rename this to something. The actual package name is `io_prometheus_client` I can rename it to promclient or something along those lines.

Reply:
renamed to promproto